session.go

```diff
@@ -595,17 +710,9 @@ func DialWithInfo(info *DialInfo) (*Session, error) {
 		}
 		session.creds = []Credential{*session.dialCred}
 	}
-	if info.PoolLimit > 0 {
-		session.poolLimit = info.PoolLimit
-	}
 
 	cluster.minPoolSize = info.MinPoolSize
```

Comment:
Why do you need cluster to have minPoolSize and maxIdleTimeMS when you have it in the DialInfo (you're using their values anyway)?

Reply:
Refactoring oversight, they're unused so I'll remove them completely.